app/models/place.rb

```diff
@@ -22,6 +22,6 @@ def self.with_enough_votes(options)
   end
 
   def image_location
-    [ENV.fetch("IMAGE_HOSTNAME", "geograph_photos"), image_uri].join("/")
+    [ENV.fetch("IMAGE_HOSTNAME", "/geograph_photos"), image_uri].join("/")
```

Comment:
Is there a situation where this fallback option helps? 

In my dev set up I needed to set `IMAGE_HOSTNAME` before I can see real photos. In live environments it sounds like we have the variable set too. 

Does some process mount photos into the app at /geograph_photos on the live environment?

Reply:
I think the situation was: the env var isn't defined in production and I don't know how to set it.

You'll have to ask Suzy or Tobias the latter question.

Reply:
I have just read [this ticket that looks relevant](https://trello.com/c/oBZBXIuX/87-speed-up-loading-of-photo-in-frame-on-homepage#comment-62faac937425e089913900e6).

Is the situation now that we need `IMAGE_HOSTNAME` in dev but no longer in prod? If so, we could try to express that notion in our code:

```ruby
# initializers/gotenv.rb
Dotenv.require_keys("IMAGE_HOSTNAME") if defined?(Dotenv)
```

```ruby
  def image_location
    uri = if Rails.env.development?
      URI(ENV["IMAGE_HOSTNAME"])
    else
      URI("/geograph_photos")
    end

    uri.path += "/" + image_uri
    uri.to_s
  end
```

Reply:
I think this is a sensible idea. I also think it doesn't need to be a blocker. Right @tahb?

Reply:
[There was another little chat on this in Slack afterwards](https://dxw.slack.com/archives/C03AET9SV45/p1660930176468789?thread_ts=1660923353.836409&cid=C03AET9SV45) where Cristina mentions it **is** used in Prod. 

I was and still am confused about how the app works. I don't think we know?

Not a blocker but it's the type of thing that will cause a headache down the road.